pkg/pod/entrypoint.go

```diff
@@ -147,15 +147,6 @@ func orderContainers(commonExtraEntrypointArgs []string, steps []corev1.Containe
 			argsForEntrypoint = append(argsForEntrypoint, resultArgument(steps, taskSpec.Results)...)
 		}
 
-		cmd, args := s.Command, s.Args
-		if len(cmd) == 0 {
-			return nil, fmt.Errorf("Step %d did not specify command", i)
-		}
```

Comment:
Did we lose any meaningful checking with this?

Reply:
Not really, at least not practically. I removed it because now it's a bit harder to tell whether there is no command, since that information might be in the env var, dependent on the runtime platform, which this code doesn't know about.

Practically speaking, if an image within a manifest list didn't specify a command or args, this PR would allow that to get down to a scheduled pod before blowing up, instead of that being caught quickly up here. In practice that's vanishingly rare (hopefully), so I don't think we really have to account for it. Unless you have some ideas?